WordPress/src/main/java/org/wordpress/android/ui/plans/PlanFragment.java

```diff
@@ -93,16 +91,16 @@ private void refreshPlanUI() {
         }
 
         // show product short name in bold, ex: "WordPress.com <b>Premium</b>"
-        TextView txtProductName = (TextView) mPlanContainerView.findViewById(R.id.text_product_name);
+        TextView txtProductName = (TextView) getView().findViewById(R.id.text_product_name);
         String productShortName = mPlanDetails.getProductNameShort();
         String productName = mPlanDetails.getProductName().replace(productShortName,
                 "<b>" + productShortName + "</b>");
         txtProductName.setText(Html.fromHtml(productName));
 
-        TextView txtTagLine = (TextView) mPlanContainerView.findViewById(R.id.text_tagline);
+        TextView txtTagLine = (TextView) getView().findViewById(R.id.text_tagline);
         txtTagLine.setText(HtmlUtils.fastUnescapeHtml(mPlanDetails.getTagline()));
 
-        // The current plan could probably have some features to highlight on the details screen
+        mPlanContainerView.removeAllViews();
```

Comment:
It seems that now that `refreshPlanUI()` is called in `onActivityCreated()` instead of `onResume()`, mPlanContainerView will always be found empty anyway. May I suggest removing this line if so? Thanks!